MaintenanceApi.yaml

```diff
@@ -5,7 +5,7 @@ info:
   description: 'FRAIHMWORK allows for maintenance to be scheduled for components. To enable this, appointments with components and duration information must be modelled in the system. The purpose of this API is to create, read, update, and delete these data. This initial version does not imply any protections against malicious behavior from trusted endpoints. This API will versioned according to SemVer 2.0.0 rules (as described here: https://semver.org/)'
   contact:
     email: matt.synborski@resilienx.com
-  version: 1.1.0
+  version: 1.1.1
```

Comment:
Should this be a minor change, since new features are added to the api?

Reply:
It should be minor because this is targeting branch 29. That branch already bumped the minor for introducing maintenance statistics. This issue just adds a few more.